src/app/settings/settings.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="choose">
+  @for(element of allCurrencies; track element.currency){
+  <button
+    (click)="setCurrency(element.currency)"
+    [class.selected]="currency() === element.currency"
+    [disabled]="!element.isAvailable && true"
+  >
+    {{ element.currency.toUpperCase() }} {{ !element.isAvailable ? "X" : "" }}
```

Comment:
Do we need the ! operator?
Can you reverse the conditions and remove the ! operator?